app/components/Calendar/index.js

```diff
@@ -0,0 +1,442 @@
+import React, { Component } from 'react';
+import FullCalendar from '@fullcalendar/react';
+import PropTypes from 'prop-types';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import momentTimezonePlugin from '@fullcalendar/moment-timezone';
+import interactionPlugin from '@fullcalendar/interaction';
+import enLocale from '@fullcalendar/core/locales/en-gb';
+import svLocale from '@fullcalendar/core/locales/sv';
+import fiLocale from '@fullcalendar/core/locales/fi';
+import '@fullcalendar/core/main.css';
+import '@fullcalendar/daygrid/main.css';
+import '@fullcalendar/timegrid/main.css';
+import moment from 'moment';
+import 'moment-timezone';
+import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import { injectIntl } from 'react-intl';
+
+import CalendarLegend from './legend/CalendarLegend';
+import {
+  isTimeRangeReservable,
+  getReservationPrice,
```

Comment:
`getReservationPrice` doesn't exist.

Reply:
Good catch. I removed the support for price, because it won't likely have many uses in this context.

Really odd that my IDE's linter did not catch this--nor did the linting scrip that the project runs on every commit.